backend/infrahub/services/adapters/workflow/local.py

```diff
@@ -10,7 +10,7 @@ async def execute(
         self,
         workflow: WorkflowDefinition | None = None,
         function: Callable[..., Awaitable[Return]] | None = None,
-        **kwargs: dict[str, Any],
+        **kwargs: Any,
```

Comment:
Why are we changing this?

Reply:
with `**kwargs: dict[str, Any],` mypy is complaining with the following message(s)  ..
```
backend/infrahub/api/schema.py:268: error: Argument "message" to "execute" of
"InfrahubWorkflow" has incompatible type "SchemaValidateMigrationData"; expected
"dict[str, Any]"  [arg-type]
    ...     workflow=SCHEMA_VALIDATE_MIGRATION, message=validate_migration_da...
```